examples/05-plotting/03-labels.py

```diff
@@ -0,0 +1,54 @@
+"""
+.. _labels:
+
+This example shows how to use nodal labels.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+One can custom labels to specific nodes with specific label properties.
+If label for a node is missing, by default nodal scalar value is shown.
+"""
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+from ansys.dpf.core.plotter import DpfPlotter
+
+model = dpf.Model(examples.msup_transient)
+
+# Get stress tensor
```

Comment:
@ayush-kumar-423 to make the example appear nicely in the documentation, could you please split your text in subparts? You can look at one of the existing example for guidelines